### PR TITLE
Aerospike Backup Configuration Changes

### PR DESCRIPTION
**Pull request by roimenashe**

### CODE REVIEW

**Review by reugn — APPROVED** [review]

File: docs/config.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ with the defined policies.
 
 ## ADR Services configurations
 
-Configuration options for each of the [ADR Services](./architecture.md#adr-services) are documented in our [Helm Chart](https://github.com/aerospike/helm-charts/tree/main/aerospike-point-in-time-recovery#configuration).
+Configuration options for each of the [ADR Services](./architecture.md#adr-services) are documented in our [Helm Charts](https://github.com/aerospike/helm-charts/tree/main/aerospike-point-in-time-recovery#configuration).
```
[the remark this answers is not in the document]
Comment (roimenashe) [reply docs/config.md:16]: I saw that most references says `helm charts`, should we use `Helm Chart` every where?

File: docs/config.md [hunk 2]
```diff
@@ -34,319 +34,416 @@ Example of a basic [aerospike.conf](https://docs.aerospike.com/server/operations
 
 # This stanza must come first.
 service {
-	user root
-	group root
-#	pidfile /var/run/aerospike/asd.pid
+    user root
+    group root
+#    pidfile /var/run/aerospike/asd.pid
```
Comment (davi17g) [inline docs/config.md:39]: Can be removed, needed only when run inside the docker

File: docs/config.md [hunk 3]
```diff
@@ -34,319 +34,416 @@ Example of a basic [aerospike.conf](https://docs.aerospike.com/server/operations
 
 # This stanza must come first.
 service {
-	user root
-	group root
-#	pidfile /var/run/aerospike/asd.pid
+    user root
+    group root
```
Comment (davi17g) [inline docs/config.md:38]: can be removed `root` by default

File: docs/config.md [hunk 4]
```diff
@@ -34,319 +34,416 @@ Example of a basic [aerospike.conf](https://docs.aerospike.com/server/operations
 
 # This stanza must come first.
 service {
-	user root
-	group root
-#	pidfile /var/run/aerospike/asd.pid
+    user root
```
Comment (davi17g) [inline docs/config.md:37]: can be removed `root` by default

**Comment by roimenashe** [comment]
@davi17g We use the setup of:
```
user root
group root
#	pidfile /var/run/aerospike/asd.pid
```

In all of the `aerospike.conf` files we have and its also the case in the official Aerospike configuration documentation: https://docs.aerospike.com/server/operations/configure.

Some of our customers would like to test on docker so pid I think is ok to keep (commented out) and user and group = root to align with the official documentation.

What do you think?

**Review by davi17g — APPROVED** [review]

**Comment by roimenashe** [comment]
> @davi17g We use the setup of:
> 
> ```
> user root
> group root
> #	pidfile /var/run/aerospike/asd.pid
> ```
> 
> In all of the `aerospike.conf` files we have and its also the case in the official Aerospike configuration documentation: https://docs.aerospike.com/server/operations/configure.
> 
> Some of our customers would like to test on docker so pid I think is ok to keep (commented out) and user and group = root to align with the official documentation.
> 
> What do you think?

David can't respond on the PR, but addressed the CR comments privately and approved the PR, merging.

**Pull request merged**